frontend/src/audio.html

```diff
@@ -30,15 +30,31 @@
     <div class="mt-1 flex flex-col justify-between md:flex-row">
       <div class="flex flex-wrap content-between align-baseline text-xs child-div:mr-1 child-div:mt-1 child-div:border-r child-div:border-gray-300 child-div:pr-1 last:child-div:border-0">
         <div>#{{ audio.objectID }}</div>
-        <div>{{ audio.dateForHumans }}</div>
-        <div>{{ audio.location }}</div>
-        <div>{{ audio.category }}</div>
+        {% if audio.dateForHumans != blank %}
```

Comment:
Why do you use `blank` check if the value should be null here actually?

https://github.com/harttle/liquidjs/issues/82

Or actually, I guess, we can just check for truthiness:
https://liquidjs.com/tutorials/truthy-and-falsy.html
https://www.11ty.dev/docs/languages/liquid/#javascript-truthiness-in-liquid